ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,458 +1,470 @@
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.neighbors[neighbor].route_map_out is defined and router_bgp.address_family_ipv4.neighbors[neighbor].route_map_out is not none %}
+{%             if router_bgp.address_family_ipv4.neighbors[neighbor].route_map_out is arista.avd.defined %}
       neighbor {{ neighbor }} route-map {{ router_bgp.address_family_ipv4.neighbors[neighbor].route_map_out }} out
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_in is defined and router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_in is not none %}
+{%             if router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_in is arista.avd.defined %}
       neighbor {{ neighbor }} prefix-list {{ router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_out is defined and router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_out is not none %}
+{%             if router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_out is arista.avd.defined %}
       neighbor {{ neighbor }} prefix-list {{ router_bgp.address_family_ipv4.neighbors[neighbor].prefix_list_out }} out
 {%             endif %}
-{%             if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate is defined %}
-      neighbor {{ neighbor }} default-originate{%if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map is defined and router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map is not none %} route-map {{ router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map }}{% endif %}{%if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.always is defined and router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.always == true %} always{% endif %}
-
+{%             if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate is arista.avd.defined %}
+{%                 set neighbor_default_originate_cli = "neighbor " ~ neighbor ~ " default-originate" %}
+{%                 if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map is arista.avd.defined %}
+{%                     set neighbor_default_originate_cli = neighbor_default_originate_cli ~ " route-map " ~ router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map %}
+{%                 endif %}
+{%                 if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.always is arista.avd.defined(true) %}
+{%                     set neighbor_default_originate_cli = neighbor_default_originate_cli ~ " always" %}
+{%                 endif %}
+      {{ neighbor_default_originate_cli }}
 {%             endif %}
 {%             if router_bgp.address_family_ipv4.neighbors[neighbor].activate == true %}
```

Comment:
No check if `router_bgp.address_family_ipv4.neighbors[neighbor].activate` is defined